libraries/libraries.mk

```diff
@@ -102,14 +102,14 @@ $(LIB_OBJECTS): INCLUDES += -I$(LIBRARIES_PATH)/features/dma
 $(LIB_OBJECTS): INCLUDES += -I$(LIBRARIES_PATH)/features/profiler
 # We should move this from AWS (and keep the license)
 $(LIB_OBJECTS): INCLUDES += -I$(AWS_FPGA_REPO_DIR)/SDAccel/userspace/include
-$(LIB_OBJECTS): CFLAGS   := -std=c11 -fPIC -D_GNU_SOURCE $(INCLUDES)
-$(LIB_OBJECTS): CXXFLAGS := -std=c++11 -fPIC -D_GNU_SOURCE $(INCLUDES)
+$(LIB_OBJECTS): CFLAGS    += -std=c11 -fPIC -D_GNU_SOURCE $(INCLUDES) -D_BSD_SOURCE
+$(LIB_OBJECTS): CXXFLAGS  += -std=c++11 -fPIC -D_GNU_SOURCE $(INCLUDES) -D_BSD_SOURCE
 # Need to move this, eventually
 #$(LIB_OBJECTS) $(PLATFORM_OBJECTS): $(BSG_MACHINE_PATH)/bsg_manycore_machine.h
 
 $(LIB_DEBUG_OBJECTS):  CXXFLAGS += -DDEBUG
 
-$(LIB_STRICT_OBJECTS): CXXFLAGS += -Wall -Werror
```

Comment:
Werror should be kept, unless there's something wrong

Reply:
Things were exploding, I'll check what specific errors there were, but there were a lot of them so I just got rid of the flag. One possible reason was I believe the default GCC was GCC8 whereas the centos GCC is GCC 4.7 or something like that.

Reply:
I can believe that...